jax/_src/numpy/lax_numpy.py

```diff
@@ -2063,6 +2063,15 @@ def atleast_3d(*arys: ArrayLike) -> Array | list[Array]:
     return [atleast_3d(arr) for arr in arys]
 
 
+def _supports_buffer_protocol(obj):
+  try:
+    view = memoryview(obj)
+  except TypeError:
+    return False
+  else:
```

Comment:
Instead of else, can you return True in the `try` block directly?

Reply:
The principle I'm following here is to put as little as possible in the `try:` block to avoid catching unexpected exceptions. But in this case, I think it's very unlikely that `return True` would raise an exception... still style-wise I marginally prefer the `else` block anyway. What do you think?

Reply:
Sure, sounds good. For me the `else` is very confusing and I have to look it up everytime ;)